RandomObjects.py

```diff
@@ -133,7 +133,10 @@ def generateSeq(self):
         # robot's up most initial y position
         robot_up = self.height / 2 + robot_radius / tile_size + 1
         goodLoc = False
-        while not goodLoc:
+        numLoops= 0
+        maxLoops= 10
```

Comment:
why is maxLoops 10?

Reply:
This was breaking we need to check it out and debug it